platform/darwin/src/MGLLight_Private.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+#import "MGLLight.h"
+
+#import <mbgl/style/light.hpp>
+#import <mbgl/style/types.hpp>
```

Comment:
Is it necessary to import types.hpp here, or is light.hpp enough? Note that it may be possible to forward-declare mbgl::style::Light instead of importing the whole header. Being judicious about includes may help us optimize Objective-C++ compilation times; I'm not sure to what extent C++ benefits from whole-module optimization.

https://github.com/mapbox/mapbox-gl-native/issues/9055#issuecomment-302821427